pokemonterminal/platform/named_event/adapters/__init__.py

```diff
@@ -0,0 +1,57 @@
+class NamedEvent:
+    """
+    Interface representing an operating system event object with a name.
```

Comment:
prefer to make this a true abstract interface using `ABC.abc` and `abstractmethod`

Reply:
I'm pretty noob at Python, I didn't knew those exist. I've made this pretty much only with Google, documentation and the existing adapter implementation. I'll replace it for that in a couple of hours.

Reply:
I'd say you're doing pretty well for python noob 😜

Reply:
Thank you :blush:

Reply:
Also, just realized (I know it's done elsewhere in this project but I find that weird too..) defining this class in the `__init__.py` is kind of weird and a bit nonbovious when looking at the files. I think it should just be in a `named_event.py` file and `__init__.py` can be empty file